ctapipe/analysis/Sensitivity.py

```diff
@@ -0,0 +1,790 @@
+from itertools import chain
+import astropy.units as u
+from astropy.table import Table
+from scipy.optimize import minimize
+import scipy.integrate as integrate
+
+import numpy as np
+
+from matplotlib import pyplot as plt
+
+
+# tau, the superior circle number
+np.tau = 2*np.pi
```

Comment:
Suggest to remove `np.tau` and instead type out `2 * np.pi` everywhere.
The problem with `np.tau` is that if I see it in the code below I'm confused because `tau` doesn't exist in numpy. So `tau = 2 * np.pi` here would be better, but for the few letters `2 * np.pi` and the few uses I don't think the global variable (which makes people scroll up and search when reading the code) is warranted.